framework/src/controller/helpers/validator/keywords/formatters.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
```

Comment:
formatting is not validation imo

Reply:
Yes but its a step covert values to right format before validation.

Reply:
I think default and formatting are still not concern of validator.
I will create a discussion issue to gather opinion and if necessary we will change it later.
Also, I will create an issue to use @liskhq/lisk-validator